src/apigility-ui/service/api.service.js

```diff
@@ -448,11 +450,13 @@
     };
 
     this.getAuthorizationRpc = function(module, version, name, callback) {
-      var moduleParam = this.normalizeModuleName(module);
+      var moduleParam = this.normalizeModuleName(module),
+          controllerName = name.replace(/-/g, '\\'),
```

Comment:
The backend understands both `-` and `\\` as separation characters within the controller service names, so this normalization isn't strictly necessary.